test/common/router/header_parser_fuzz_test.cc

```diff
@@ -9,11 +10,32 @@ namespace Envoy {
 namespace Fuzz {
 namespace {
 
+using Envoy::replaceInvalidCharacters;
+
+// Remove invalid characters from HeaderValueOptions in headers to add.
+Protobuf::RepeatedPtrField<envoy::api::v2::core::HeaderValueOption> processHeadersToAdd(
```

Comment:
Should this helper be in https://github.com/envoyproxy/envoy/blob/master/test/fuzz/utility.h? That is ideally where we can centralize the `Header` message style utils.

Reply:
Yes!! I didn't see that. Honestly, I think I should move the function in https://github.com/envoyproxy/envoy/blob/master/test/common/http/common_fuzz.h here as well.

Reply:
Sure, makes sense.